gms/gossiper.cc

```diff
@@ -783,31 +780,6 @@ void gossiper::run() {
                     logger.trace("Faill to do_gossip_to_unreachable_member: {}", ep);
                 });
 
-                /* Gossip to a seed if we did not do so above, or we have seen less nodes
-                   than there are seeds.  This prevents partitions where each group of nodes
-                   is only gossiping to a subset of the seeds.
-
```

Comment:
How do we avoid this problem?

I don't think it's a problem that we really need to avoid, but let's make sure.

Reply:
We avoid this problem by contacting all initial contact point nodes (seeds node currently).

Reply:
Ah, this is just during startup. And we'll move to sending to all peers (if not bootstrapping), yes?

Reply:
Yes.